joda-time/src/main/java/org/joda/time/field/PreciseDateTimeField.java

```diff
@@ -77,13 +77,14 @@ public PreciseDateTimeField(DateTimeFieldType type,
      * @return the amount of fractional units extracted from the input.
      */
     @Override
-    public int get(long instant) {
-        if (instant >= 0) {
-            return (int) ((instant / getUnitMillis()) % iRange);
-        } else {
-            return iRange - 1 + (int) (((instant + 1) / getUnitMillis()) % iRange);
+        public int get(long instant) {
```

Comment:
This `public` is indented too much. See below (or above) for other methods.

Reply:
I apologize for my overlook for some of these mistakes, I fixed this now and committed the code